src/renderers/WebGLRenderer.js

```diff
@@ -653,24 +658,24 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	var deallocateTexture = function ( texture ) {
 
-		if ( texture.image && texture.image.__webglTextureCube ) {
+		if ( texture.image && objectRendererWebGLProps[texture.uuid].__image__webglTextureCube ) {
```

Comment:
Added checks at the beginning of functions in which the webgl properties are accessed.

Another strategy would be to encapsulate access of webgl properties with a getter and setter, (like what's going on in webglObject with attribute buffers).

It would looks something like"

```javascript

function getWebGLProps (obj) {
   if ( ! objectRendererWebGLProps[object.uuid]) {
    
        objectRendererWebGLProps[object.uuid] = {};

   }

   return objectRendererWebGLProps[object.uuid];
}

getWebGLProps(texture)
```

Seems better design, but I don't think the extra condition / lookup is worth it.